macros/rename_partition_tstamp_date.sql

```diff
@@ -0,0 +1,14 @@
+{% macro rename_partition_tstamp_date() %}
```

Comment:
Can we add it to the schema.yml macro definitions? Just so that everyone is clear that it is to work around dbt compilation issue to update config based on a var.

Reply:
Done